ShellPkg/Application/Shell/Shell.c

```diff
@@ -575,6 +575,13 @@ UefiMain (
 
     Size       = 100;
     TempString = AllocateZeroPool (Size);
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (TempString == NULL) {
+      ASSERT (TempString != NULL);

```

Comment:
nit: we didn't assert before, do we want the assert now? Probably fine as before we would have null pointer dereferenced.

Reply:
I think we should assert here. If this allocation fails, someone should be aware a 100 byte allocation is not possible.